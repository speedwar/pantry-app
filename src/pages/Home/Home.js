import React from 'react'
import { BrowserRouter } from 'react-router-dom'
import { Provider } from 'react-redux'
import { store } from 'rx/store'
import PmInventory from 'components/PmInventory/PmInventory'

import 'sass/main.scss'

const Content = () => (
  <div className="pm-app">
    <main className="container">
      <PmInventory />
    </main>
  </div>
)

const App = () => (
  <Provider store={ store }>
    <BrowserRouter>
      <Content />
    </BrowserRouter>
  </Provider>
)

export default App
