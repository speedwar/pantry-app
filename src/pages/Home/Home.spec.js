import React from 'react'
import { shallow } from 'enzyme'
import Home from './Home'

const setup = () => {
  const wrapper = shallow(<Home />)
  return { wrapper }
}

describe('<Home /> component', () => {
  it('renders component', () => {
    const wrapper = setup()
    expect(wrapper).toMatchSnapshot()
  })
})
