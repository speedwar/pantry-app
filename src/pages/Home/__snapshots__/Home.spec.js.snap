// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> component renders component 1`] = `
Object {
  "wrapper": <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <BrowserRouter>
      <Content />
    </BrowserRouter>
  </Provider>,
}
`;
